### PR TITLE
New video feature

### PR DESCRIPTION
**Pull request by up-87**
#### Prerequisites checklist

- [X] I have read the [contributing guidelines](https://github.com/PhotoboothProject/photobooth/blob/dev/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "x" next to an item)

- [ ] Documentation update
- [ ] Bug fix
- [X] New feature
- [ ] Other, please explain:

#### What changes did you make? (Give an overview)
New video feature. (Includes changes in branch feature/improvements-from-video)

### CODE REVIEW

File: api/previewCamera.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<?php
```
Comment (andi34) [inline api/previewCamera.php]: Leftover. Was moved to https://github.com/PhotoboothProject/photobooth/pull/69.

File: lib/collage.php [hunk 2]
```diff
@@ -1,42 +1,17 @@
 <?php
 require_once __DIR__ . '/config.php';
+require_once __DIR__ . '/collageConfig.php';
```
Comment (andi34) [inline lib/collage.php]: Leftover. Moved to https://github.com/PhotoboothProject/photobooth/pull/69.

File: lib/collageConfig.php [hunk 3]
```diff
@@ -0,0 +1,117 @@
+<?php
```
Comment (andi34) [inline lib/collageConfig.php]: Same here

File: src/js/preview.js [hunk 4]
```diff
@@ -92,7 +92,7 @@ const photoboothPreview = (function () {
         };
 
         jQuery
-            .post('api/takeVideo.php', dataVideo)
+            .post('api/previewCamera.php', dataVideo)
```
Comment (andi34) [inline src/js/preview.js]: Sam's here

File: api/previewCamera.php (hunk 1 above)
Comment (up-87) [reply api/previewCamera.php]: Those are shown here because both MR are to dev. As this branch is based on #69 all changes from that are in here, as well.

**Comment by andi34** [comment]
- [X] Rebased.
- [X] Fixed conflicts
- [X] Video icon adjustable

**Pull request merged**